var/spack/repos/builtin/packages/stata/package.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+import llnl.util.filesystem as fs
+from datetime import datetime
+
+
+class Stata(Package):
+    """STATA is a general-purpose statistical software package developed
+       by StataCorp."""
+
+# Known limitations of this installer:
+# * STATA requires libpng v12 that ships with EL enterprise distros. Spack
+#   provides version 16 with a "depends_on('libpng')" and STATA checks
+#   explicitly for 12. So until 12 is packaged, `yum install libpng`. :-)
```

Comment:
Shouldn't be too hard to add `libpng@12` to Spack and then explicitly `depends_on('libpng@12.0:12.999')`. Never rely on the system package manager.

Reply:
I missed it when I was looking earlier but the libpng package _does_ provide libpng 1.2.57. Hooray! Just pulled it in and it works! Will have this done in this next commit. Thanks!